### PR TITLE
exui-1316

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/EXUI-1294
https://tools.hmcts.net/jira/browse/EXUI-1295
https://tools.hmcts.net/jira/browse/EXUI-1316

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by OgunyemiO — DISMISSED** [review]

**Review by OgunyemiO — APPROVED** [review]

**Review by andywilkinshmcts — DISMISSED** [review]

**Review by andywilkinshmcts — DISMISSED** [review]

File: src/hearings/containers/request-hearing/hearing-judge/hearing-judge.component.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { JudicialRefDataService } from '../../../services/judicial-ref-data.serv
 import * as fromHearingStore from '../../../store';
 import { ValidatorsUtils } from '../../../utils/validators.utils';
 import { RequestHearingPageFlow } from '../request-hearing.page.flow';
+import { HearingsUtils } from 'src/hearings/utils/hearings.utils';
```
Comment (connorpgpmcelroy) [inline src/hearings/containers/request-hearing/hearing-judge/hearing-judge.component.ts:16]: src

**Review by kasi-subramaniam — APPROVED** [review]

File: src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ import { PanelPreferenceModel } from '../../../models/panelPreference.model';
 import { HearingsService } from '../../../services/hearings.service';
 import * as fromHearingStore from '../../../store';
 import { RequestHearingPageFlow } from '../request-hearing.page.flow';
+import { HearingsUtils } from 'src/hearings/utils/hearings.utils';
```
Comment (connorpgpmcelroy) [inline src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.ts:22]: src

File: src/hearings/converters/hearing-panel.amended.converter.spec.ts [hunk 3]
```diff
@@ -43,20 +43,32 @@ describe('HearingPanelAmendedConverter', () => {
...
-  it('should not transform the amended flag when previous vs current hearing type are equal', () => {
+  it('should not transform the amended flag to false when previous vs current hearing type are equal', () => {
     const STATE: State = _.cloneDeep(initialState.hearings);
-    STATE.hearingRequest.hearingRequestMainModel.hearingDetails.panelRequirements = null;
+    STATE.hearingRequest.hearingRequestMainModel.hearingDetails.panelRequirements = {
+      panelPreferences: JUDICAIL_USER_DETAILS,
+      roleType: ['role1']
+    };
+    STATE.hearingRequestToCompare.hearingRequestMainModel.hearingDetails.panelRequirements = {
+      panelPreferences: JUDICAIL_USER_DETAILS,
```
Comment (connorpgpmcelroy) [inline src/hearings/converters/hearing-panel.amended.converter.spec.ts:69]: minor comment - not related to changes. JUDICIAL not JUDICAIL

**Review by connorpgpmcelroy — APPROVED** [review]
Approved with minor comments

**Pull request merged**